core/src/main/java/feast/core/service/SpecService.java

```diff
@@ -94,8 +95,9 @@ public GetFeatureSetResponse getFeatureSet(GetFeatureSetRequest request)
     if (request.getName().isEmpty()) {
       throw new IllegalArgumentException("No feature set name provided");
     }
+    // Autofill default project if project is not specified
     if (request.getProject().isEmpty()) {
-      throw new IllegalArgumentException("No project provided");
+      request = request.toBuilder().setProject("default").build();
```

Comment:
Should this be using one global project constant?

Reply:
Corrected.